doc/source/fault-tolerance.rst

```diff
@@ -46,12 +46,13 @@ Actors
 ------
 
 If an actor process crashes unexpectedly, Ray will attempt to reconstruct the
-actor process up to a maximum number of times. This value can be specified with
-the ``max_reconstructions`` keyword, which by default is ``0``. If the maximum
-number of reconstructions has been used up, then subsequent actor methods will
+actor process. The desired functionality can be specified with
```

Comment:
Try to be as concise as possible in documentation. This block could be rephrased as:

"Ray will automatically restart actors that crash unexpectedly. This behavior is controlled using `max_restarts`, which sets the maximum number of times that an actor will be restarted. If 0, the actor won't be restarted. If -1, it will be restarted infinitely."

Reply:
Done.